src/sentry/models/commitfilechange.py

```diff
@@ -31,6 +31,7 @@ class CommitFileChange(Model):
     type = models.CharField(
         max_length=1, choices=(("A", "Added"), ("D", "Deleted"), ("M", "Modified"))
     )
+    language = models.CharField(default=None, null=True, max_length=255)
```

Comment:
If we take out `default=None` here what happens? Since we have `null=True` will it default to `None` anyway?

Reply:
Yes the default default value is `None` for nullable fields.